### DISCUSSION TITLE
Solution for adding durability to some indexes?

### DISCUSSION THREAD

**Discussion by lemasteralan** [Security-Onion-Solutions/securityonion#2900] [Unsupported Versions] [answered]
Is there a common solution for adding durability to some indexes in elasticsearch in the default Security Onion install architecture for v2?

I understand from the documentation that the philosophy is to deliver "best effort" service with network security monitoring. I'm looking at the scenario of leveraging the elasticsearch clustering included with Security Onion for some additional security use cases with fairly long time horizons (years). I'd like to try to provide some additional durability to those indexes beyond the default. I'm not trying to do full backups of every index or pcap data.

I'm looking for a solution to avoid data loss in some indexes if we have to recover a single search node. With the default cross-cluster architecture I'm not aware of any way to replicate an index to another search node on a free license. I believe the elasticsearch native solution would be cross cluster replication, which would require a platinum license subscription.

Is there a known solution for this with the default architecture in the most recent release (2.3)? Do you see the traditional elasticsearch clustering as a partial answer to this?

**Comment by TOoSmOotH** [comment, marked as answer]
Traditional clustering does not require a license. Our architecture is like it is to be simple. Large clusters can become a full time job when it comes to shard management. However if you need the full redundancy that true clustering provides, you can select "Advanced" during the manager install and then "Yes" to traditional ES clustering. Replicas can be set in the global.sls file in the local pillar directory.
